pkg/controller/extension/actuator_test.go

```diff
@@ -247,6 +250,52 @@ var _ = Describe("GenerateInstallManifest", func() {
 	})
 })
 
+var _ = Describe("ReconcileShootInfoConfigMap", func() {
+	var (
+		shootClient client.Client
+		config      *fluxv1alpha1.FluxConfig
+	)
+
+	It("should apply successfully and contain expected keys", func() {
+		shootName := "test-shoot"
+		technicalID := fmt.Sprintf("shoot--asdf-test-%s", shootName)
```

Comment:
Nit: the technical ID for new shoots always has two dashes as separators. You can also use the corresponding helper function (`gardenerutils.ComputeTechnicalID`) if you want.
```suggestion
		technicalID := fmt.Sprintf("shoot--asdf-test--%s", shootName)
```